sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -375,42 +461,81 @@ class TopicRuntimeInfo(object):
     :ivar accessed_at: Last time a message was sent, or the last time there was a receive request
      to this queue.
     :type accessed_at: ~datetime.datetime
-    :ivar message_count_details: Details about the message counts in queue.
-    :type message_count_details: ~azure.servicebus.management._generated.models.MessageCountDetails
     :ivar subscription_count: The number of subscriptions in the topic.
     :type subscription_count: int
+    :ivar active_message_count: Number of active messages in the queue, topic, or subscription.
+    :type active_message_count: int
+    :ivar dead_letter_message_count: Number of messages that are dead lettered.
+    :type dead_letter_message_count: int
+    :ivar scheduled_message_count: Number of scheduled messages.
+    :type scheduled_message_count: int
+    :ivar transfer_dead_letter_message_count: Number of messages transferred into dead letters.
+    :type transfer_dead_letter_message_count: int
+    :ivar transfer_message_count: Number of messages transferred to another queue, topic, or
+     subscription.
+    :type transfer_message_count: int
     """
     def __init__(
         self,
-        name,
-        **kwargs
     ):
-        # type: (str, Any) -> None
-        self.name = name
+        self._name = None
         self._internal_td = None  # type: Optional[InternalTopicDescription]
-        self.created_at = kwargs.get('created_at', None)
-        self.updated_at = kwargs.get('updated_at', None)
-        self.accessed_at = kwargs.get('accessed_at', None)
-        self.message_count_details = kwargs.get('message_count_details', None)
-        self.subscription_count = kwargs.get('subscription_count', None)
 
     @classmethod
     def _from_internal_entity(cls, name, internal_td):
-        # type: (str, InternalTopicDescription) -> TopicRuntimeInfo
-        qd = cls(name)
+        # type: (str, InternalTopicDescription) -> TopicRuntimeProperties
+        qd = cls()
+        qd._name = name
         qd._internal_td = internal_td
```

Comment:
in TopicProperties, deepcopy is used, `qd._internal_td = deepcopy(internal_td)`, here not

Reply:
Runtime properties doesn't need deepcopy because all properties of this class are read-only